static/js/components/ProgressWidget.js

```diff
@@ -142,7 +146,8 @@ export default class ProgressWidget extends React.Component {
             60,
             6,
             totalPassedCourses,
-            program.number_courses_required
+            program.number_courses_required,
+            program.program_letter_url
```

Comment:
Why are we adding it here? 7 lines below we also have the program letter. Are we duplicating it?

Reply:
It doesn't pick the program_letter_url if we don't pass it here

Reply:
It only shows the link on the learner page now because the previous code doesn't show any link on it

Reply:
If you want to add the letter to the circularProgressWidget then you have to remove this code below:
https://github.com/mitodl/micromasters/blob/7bb555db606da884dece82ba2d00010fe949ce7b/static/js/components/ProgressWidget.js#L150-L153
Otherwise it is duplicated.